modules/serverless-rn-1-29-1.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies
+//
+// * /serverless/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-29-1_{context}"]
+= Release notes for Red Hat {ServerlessProductName} 1.29.1
+
+{ServerlessProductName} 1.29.1 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+
+[id="new-features-1-29-1_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.8.
+* {ServerlessProductName} now uses Knative Eventing 1.8.
+* {ServerlessProductName} now uses Kourier 1.8.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.8.1.
+* {ServerlessProductName} now uses Knative Kafka 1.8.0.
+* The `kn func` CLI plug-in now uses `func` 1.10. FIXME is this correct?
+
+* This release of {ServerlessProductName} addresses Common Vulnerabilities and Exposures (CVEs), contains bug fixes, and is supported on OpenShift Container Platform 4.10 and later versions.
+
+[id="known-issues-1-29-1_{context}"]
+== Known issues
+
+* The `net-kourier-controller` might not be able to start due to the liveness probe error. You can work around the problem using the Knowledgebase solution.
```

Comment:
This issue was actually fixed in 1.29.1. We verified that: https://issues.redhat.com/browse/SRVKS-1078
It should be in "Fixed issues" section and we don't need to reference the knowledge base article.